tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/UpdateCombinationFeatureContext.php

```diff
@@ -138,7 +138,7 @@ private function fillCommand(UpdateCombinationCommand $command, array $dataRows)
         }
         // References
         if (isset($dataRows['ean13'])) {
-            $command->setEan13($dataRows['ean13']);
+            $command->setGtin($dataRows['ean13']);
```

Comment:
Not blocking, but it would have been nice if e update the behat feature scenarios to update this column name If you don't think it's relevant it's fine as well